app/views/themes/cultural_show/hyrax/base/show.html.erb

```diff
@@ -12,26 +12,29 @@
         <div class="row">
           <%= render 'workflow_actions_widget', presenter: @presenter %>
           <% if @presenter.universal_viewer? %>
-            <div class="col-sm-12">
+            <div class="col-sm-12 centered-media">
               <%= render 'representative_media', presenter: @presenter, viewer: true %>
             </div>
           <% end %>
           <div class="col-sm-12">
             <%= render "show_actions", presenter: @presenter %>
           </div>
-          <div class="<%= !@presenter.universal_viewer? ? 'col-sm-3 text-center' : '' %>">
+          <div class="centered-media <%= !@presenter.universal_viewer? ? 'col-sm-6 text-center' : '' %>">
             <%= render 'representative_media', presenter: @presenter, viewer: false unless @presenter.universal_viewer? %>
           </div>
-          <div class="<%= @presenter.universal_viewer? ? 'col-sm-8' : 'col-sm-6' %>">
+          <div class="<%= @presenter.universal_viewer? ? 'col-sm-6' : 'col-sm-6' %>">
             <%= render 'work_description', presenter: @presenter %>
             <%= render 'metadata', presenter: @presenter %>
           </div>
-          <div class="<%= @presenter.universal_viewer? ? 'col-sm-4' : 'col-sm-3' %>">
+          <div class="col-xs-12 relationships-container">
             <div class='relationships-box'>
               <%= render 'relationships', presenter: @presenter %>
+              <% if @presenter.class == Hyrax::OerPresenter %>
```

Comment:
OER is a custom worktype for PALNI PALCI code base, so this should be removed (lines 32-34)